miner/README.org

```diff
@@ -25,24 +25,24 @@
 
 * What is Mining?
 
-A blockchain is a series of /blocks/. Blocks contain /transactions/ which
+A blockchain is a series of /blocks/. Blocks contain /transactions/ that
 represent coin transfers or other Smart Contract interactions. Mining is the
 process of mathematically "solving" a block. Unless solved, a block cannot be
 included in the chain. Without mining, a blockchain cannot progress and
 transactions will never finalize.
 
-Economically, mining is the principal way to obtain currency on a Chainweb
-network. Since Chainweb uses Proof-of-Work, computational effort is rewarded
+Economically, mining is the principal way to obtain currency on a Kadena
+network. Since Kadena uses Proof-of-Work, computational effort is rewarded
 with coins associated with each block you solve. The more computational power
 you have, the more likely you are to be successful at mining.
 
-Unlike blockchains that have a single chain, Chainweb has many. Everyone who
+Unlike blockchains that have a single chain, the Kadena public blockchain has many. Everyone who
 chooses to mine can spread their effort across multiple chains, advancing each
 chain equally, thereby reducing competition and wasted effort.
 
-* Chainweb Mining via Official Kadena Software
+* Kadena Mining via Official Kadena Software
```

Comment:
We could just remove the name here altogether, and make it:

> Mining via Official Kadena Software

Where here, "Kadena" refers to the company.

Reply:
Love it. Let's move forward with that change.